Before: b23a60f243c439d9316eab179162b22e7506fc2a61fbd3c8526f3edb7eb197b0
After: 42d2ac257efa9e319132158f2db5ea29e9971aab5fd3ec1b0cc365b679b58d99
Patch: 1dfb6a07c7d02da746e98b1651a108ae831681b8e1291d5dd5af6996f899cf3d
--- corvid-features/plugin.rb	2012-08-19 10:35:11.405867101 +0530
+++ corvid-features/plugin.rb	2012-08-19 10:47:49.257028686 +0530
@@ -1,5 +1,3 @@
-requirements 'corvid:corvid'
-
 install {
   empty_directory 'resources'
   empty_directory 'resources/latest'
--- corvid-features/test_spec.rb	2012-08-19 10:35:11.832530417 +0530
+++ corvid-features/test_spec.rb	2012-08-19 10:47:49.257028686 +0530
@@ -1,5 +1,3 @@
-requirements 'corvid:corvid'
-
 install {
   copy_file               '.rspec'
   copy_executable         'bin/guard'
--- corvid-features/test_unit.rb	2012-08-19 10:35:11.865863321 +0530
+++ corvid-features/test_unit.rb	2012-08-19 10:47:49.257028686 +0530
@@ -1,5 +1,3 @@
-requirements 'corvid:corvid'
-
 install {
   copy_executable         'bin/guard'
   copy_file_unless_exists 'test/bootstrap/all.rb'
--- resources/latest/corvid-features/%name%.rb.tt	2012-08-19 10:47:06.330673814 +0530
+++ resources/latest/corvid-features/%name%.rb.tt	2012-08-19 10:47:49.227028903 +0530
@@ -1,10 +1,3 @@
-# Requirements that must be satisfied before the feature can be installed.
-# @return [nil, String, Hash<String,Fixnum|Range|Array<Fixnum>>, Array] Requirements that can be provided to
-#   {RequirementValidator}.
-# @see RequirementValidator#add
-#
-#requirements
-
 install {
   # empty_directory 'lib'
   # copy_file       'hello.txt'
